scripts/publish.sh

```diff
@@ -91,6 +87,19 @@ ARCHITECTURES=("amd64" "arm64")
 # This variable is used in the image tag
 init="init"
 
+# Sets AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB variable with the latest version available in dockerhub.
+get_latest_dockerhub_version() {
+    username="$(aws secretsmanager get-secret-value --secret-id $DOCKER_HUB_SECRET --region ${1} | jq -r '.SecretString | fromjson.username')"
+    password="$(aws secretsmanager get-secret-value --secret-id $DOCKER_HUB_SECRET --region ${1} | jq -r '.SecretString | fromjson.password')"
+    payload="{\"username\": \"${username}\",\"password\": \"${password}\"}"
+    token=$(curl -s -X POST -H "Content-Type: application/json" -d "${payload}"  https://hub.docker.com/v2/users/login | jq -r '.token')
+    auth="Authorization: Bearer ${token}"
+    docker_hub_image_tags=$(curl -s -S -H "${auth}" 'https://registry.hub.docker.com/v2/repositories/amazon/aws-for-fluent-bit/tags/?page=1&page_size=250' | jq -r '.results[].name')
+    tag_array=(`echo ${docker_hub_image_tags}`)
+    AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB=$(./get_latest_dockerhub_version.py ${tag_array[@]})
```

Comment:
This script doesn't need to be authorized too, does it?

Reply:
Oh i see. It's just a parser. https://github.com/PettitWesley/aws-for-fluent-bit/blob/auth-to-dockerhub/scripts/get_latest_dockerhub_version.py